package.json

```diff
@@ -2,10 +2,23 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "test": "make test",
-    "build": "make build",
-    "lint": "make lint",
-    "fix": "make fix"
+    "build": "npm run clean && gulp build",
+    "build-dist": "npm run build && node packages/babel-polyfill/scripts/build-dist.js && node packages/babel-runtime/scripts/build-dist.js && node scripts/generate-babel-types-docs.js",
+    "watch": "npm run clean && gulp watch",
+    "lint": "eslint packages/*/{src,test}/*.js --format=codeframe",
+    "fix": "eslint packages/*/{src,test}/*.js --format=codeframe --fix",
+    "flow": "flow check",
+    "clean": "npm run test-clean && rimraf packages/*/lib packages/babel-polyfill/browser* packages/babel-polyfill/dist coverage packages/*/npm-debug*",
+    "test-clean": "npm update rimraf && rimraf packages/*/test/tmp packages/*/test-fixtures.json",
+    "clean-all": "npm run clean && rimraf packages/*/node_modules \"node_modules/!(rimraf)\" \"node_modules/.bin/!(rimraf*)\" && npm uninstall rimraf",
+    "test-only": "npm run tests && npm run test-clean",
+    "test": "npm run lint && npm run test-only",
+    "test-cov": "npm run clean && rimraf packages/*/lib && cross-env BABEL_ENV=test gulp build && npm run cov-tests",
+    "tests": "cross-env NODE_ENV=test scripts/node-run-tests.js",
+    "cov-tests": "cross-env NODE_ENV=test node node_modules/istanbul/lib/cli.js cover scripts/run-tests.js -- --color || true",
+    "test-ci": "npm run bootstrap && npm run cov-tests && node node_modules/codecov.io/bin/codecov.io.js < coverage/coverage.json",
+    "publish": "git pull --rebase && rimraf packages/*/lib && cross-env BABEL_ENV=production npm run build-dist && npm test && lerna publish --only-explicit-updates && npm run clean",
+    "bootstrap": "npm run clean-all && npm install && lerna bootstrap && npm run build && cd packages/babel-runtime && npm install && node scripts/build-dist.js"
```

Comment:
To me it seems like this would be way cleaner if you used a task runner like Gulp. All the `npm run` and `&&` are purely syntax noise and Gulp's syntax for describing dependencies between tasks is much nicer.